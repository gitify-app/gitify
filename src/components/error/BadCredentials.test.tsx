import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';

import { BadCredentials } from './BadCredentials';

describe('components/error/BadCredentials.tsx', function () {
  it('should render itself & its children', function () {
    const tree = TestRenderer.create(<BadCredentials />);

    expect(tree).toMatchSnapshot();
  });
});
