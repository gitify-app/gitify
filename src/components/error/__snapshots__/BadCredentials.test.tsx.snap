// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/error/BadCredentials.tsx should render itself & its children 1`] = `
<div
  className="flex flex-1 flex-col justify-center items-center p-4 bg-white dark:bg-gray-dark text-black dark:text-white"
>
  <h1
    className="text-5xl mb-5"
  >
    ðŸ”“
  </h1>
  <h2
    className="font-semibold text-xl mb-2 text-semibold"
  >
    Bad Credentials
  </h2>
  <div>
    The credentials you are may be invalid or expired
  </div>
</div>
`;
